14/events/index.html

```diff
@@ -0,0 +1,1234 @@
+    var tempHTML =  container.innerHTML;
+    container.innerHTML = description[eventName];
+    description[eventName] = tempHTML;
+    }
+    </script>
+
+    <title>Events</title>
+</head>
+
+<body style="background-color:rgba(0,0,0,0.5);width:95%;">
+    <div class="container">
+        <div class="ca-container" id="ca-container">
+            <div class="ca-wrapper">
+                <div class="ca-item ca-item-1">
+                    <div class="ca-item-main">
+                        <a class="ca-more-icon" href="#"></a>
+
+                        <div class="ca-icon">
+                        </div>
+
+
+                        <h3>Threads</h3>
+                        <a href="http://www.qualcomm.co.in/" target=
+                        "_blank"><img alt="Qualcomm" height="50px" src=
+                        "../sponsors/img/sponsors/b_qualcomm.jpg" width=
+                        "100px"></a><br>
+                        <a class="link" href=
+                        "http://felicity.iiit.ac.in/threads/" target=
+                        "_blank">Click here for HomePage</a><br>
+                        <br>
+                        <span>Because to iterate is human, to recurse is
+                        divine</span> <a class="ca-more" href="#" id="link"
+                        name="link">more...</a>
+                    </div>
+
+
+                    <div class="ca-content-wrapper">
+                        <div class="ca-content">
+                            <a class="ca-close" href="#">close</a>
+
+                            <div class="ca-content-text" id="threads">
+                                <table>
+                                    <tr>
+                                        <td>
+                                            <h6>Online</h6>
+
+
+                                            <ul style="font-size:20px">
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href=
+                                                "http://felicity.iiit.ac.in/threads/events/codecraft"
+                                                style="width:100px" target=
+                                                "_blank">Code-Craft</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href=
+                                                "http://felicity.iiit.ac.in/threads/events/tle"
```

Comment:
LOL.

Nope. :grin: 

It is now `https://felicity.iiit.ac.in/threads/events/tle`